code/src/components/Wrapper.js

```diff
@@ -0,0 +1,75 @@
+import React, { useEffect, useState } from "react";
+
+import Thoughts from "./Thoughts";
+import Form from "./Form";
+import Loading from "./Loading";
+
+import { API_URL, LIKES_URL } from "utils/urls";
+
+const Wrapper = () => {
```

Comment:
Using a wrapper is a good idea. It's a cleaner approach than putting everything into App.js.